Formula/imessage-exporter.rb

```diff
@@ -0,0 +1,22 @@
+class ImessageExporter < Formula
+  desc "iMessage conversation database export and inspector"
+  homepage "https://github.com/ReagentX/imessage-exporter"
+  url "https://github.com/ReagentX/imessage-exporter/archive/refs/tags/1.1.0.tar.gz"
+  sha256 "b932862b5f4c1c607c4dd401f99d088b4e1291ce7d5c2a7d1c40da9c91478e97"
+  license "GPL-3"
```

Comment:
Can it use later versions?

Reply:
I don't believe so; don't see "or later" specified anywhere, unless I'm misunderstanding and that's the default? https://github.com/ReagentX/imessage-exporter/blob/develop/LICENSE